kontent-ai/integration-zapier-typescript
# Fix linked items element input

### Motivation

Fixes incorrect handling of list types when upserting elements (e.g. linked items, taxonomy).

### Checklist

- [x] Code has been tested in a private Zapier integration

### How to test

You can check that inserting multiple linked items or other list-type elements work correctly both as a Zapier "native" list and as a single element where the elements are separated in a string with ",".
